### PR TITLE
feat(core): Update default CAS URL to point to clay infra

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Comment by stbrody** [comment]
@valmack, we should try to make sure that that URL is valid and usable before releasing this, even if it's still pointing to old infra and needs to be redirected shortly thereafter.

File: packages/cli/src/ceramic-daemon.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const DEFAULT_PORT = 7007
 const DEFAULT_NETWORK = 'testnet-clay'
 const toApiPath = (ending: string): string => '/api/v0' + ending
 
-const DEFAULT_ANCHOR_SERVICE_URL = "https://cas.3box.io:8081"
+const DEFAULT_ANCHOR_SERVICE_URL = "https://cas-clay.3box.io:8081"
```
Comment (valmack) [inline packages/cli/src/ceramic-daemon.ts:14]: should be `https://cas-clay.3boxlabs.com`

**Review by valmack — COMMENTED** [review]
Will point it to dev until new one is yp

File: packages/cli/src/ceramic-daemon.ts (hunk 1 above)
Comment (stbrody) [reply packages/cli/src/ceramic-daemon.ts]: Done

**Review by valmack — APPROVED** [review]

**Pull request merged**